src/frontend/src/components/RecordSearch/Record/RecordSummary/ChargesList.tsx

```diff
@@ -13,20 +13,26 @@ export default class ChargesList extends React.Component<Props> {
       const chargesNames = chargeGroup[1];
       const listItems = this.buildListItems(chargesNames);
       const labelColor = (eligibilityDate==="Eligible now" ? "green" : eligibilityDate==="Ineligible" ? "red" : eligibilityDate==="Need more analysis" ? "purple" : "dark-blue");
-      return (
-        <div key={index}>
-          <div className="mb1">
-            <span className={"fw7 ttc mb2 " + labelColor}> {eligibilityDate} </span> <span>{(chargesNames.length > 0 ? "(" + chargesNames.length + ")" : "" )}</span>
+      if (eligibilityDate==="Need more analysis" && listItems == "None") {
```

Comment:
Probably adjust `buildListItems` so that `listItems == "None"` can be replaced with something like `listItems.length === 0`.